### PR TITLE
Add support of displaying nested data structures of Kafka in the right-pane

### PR DESCRIPTION
**Pull request by mertyildiran**
Ignore the unresolved `src.name` and `dst.name` fields. It's because I'm running it locally.

### Before:

Screenshot from 2022-01-14 19-07-26

### After:

Screenshot from 2022-01-14 21-13-26

### Before

Screenshot from 2022-01-14 20-08-37

### After

Screenshot from 2022-01-14 21-13-35

### Before

Screenshot from 2022-01-14 20-02-51

### After

Screenshot from 2022-01-14 21-13-58

### Before

Screenshot from 2022-01-14 20-02-52

### After

Screenshot from 2022-01-14 21-14-02

### Before

Screenshot from 2022-01-14 20-19-48

### After

Screenshot from 2022-01-14 21-14-23

### Before

Screenshot from 2022-01-14 20-19-55

### After

Screenshot from 2022-01-14 21-14-38

### CODE REVIEW

File: ui/src/components/EntryDetailed/EntrySections.tsx [hunk 1]
```diff
@@ -167,7 +169,7 @@ export const EntryBodySection: React.FC<EntryBodySectionProps> = ({
 
     return <React.Fragment>
         {content && content?.length > 0 && <EntrySectionContainer
-                                                title='Body'
+                                                title={title}
```
Comment (gadotroee) [inline ui/src/components/EntryDetailed/EntrySections.tsx:172]: this is the same change the you made in https://github.com/up9inc/mizu/pull/644, right?
if you merge the redis one before this will not be part of this change IMO

File: ui/src/components/EntryDetailed/EntryViewer.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const SectionsRepresentation: React.FC<any> = ({data, color, updateQuery}) => {
                     break;
                 case SectionTypes.SectionBody:
                     sections.push(
-                        <EntryBodySection key={i} color={color} content={row.data} updateQuery={updateQuery} encoding={row.encoding} contentType={row.mimeType} selector={row.selector}/>
+                        <EntryBodySection key={i} title={row.title} color={color} content={row.data} updateQuery={updateQuery} encoding={row.encoding} contentType={row.mimeType} selector={row.selector}/>
```
Comment (gadotroee) [inline ui/src/components/EntryDetailed/EntryViewer.tsx:24]: same as previous message about this change as part of https://github.com/up9inc/mizu/pull/644

File: ui/src/components/EntryDetailed/EntrySections.tsx (hunk 1 above)
Comment (mertyildiran) [reply ui/src/components/EntryDetailed/EntrySections.tsx:172]: @gadotroee doesn't matter. It required the same thing. We'll resolve the conflicts if any.

File: ui/src/components/EntryDetailed/EntryViewer.tsx (hunk 2 above)
Comment (mertyildiran) [reply ui/src/components/EntryDetailed/EntryViewer.tsx:24]: @gadotroee doesn't matter. It required the same thing. We'll resolve the conflicts if any.

**Review by gadotroee — APPROVED** [review]
